### DISCUSSION TITLE
Reopt API is returning 500 - 2023-01-03

### DISCUSSION THREAD

**Discussion by vtnate** [NREL/REopt-API-Analysis#95] [2 - Site & Utility Questions] [answered]
Is this the correct place to check for and notify about reopt down status?

**Comment by LParkhill** [comment, marked as answer]
@vtnate, yes the notification the REopt API is down should have been posted here. Putting it up now. Thanks!

**Reply by LParkhill** [reply]
@vtnate, REopt is back up. Sorry for your inconvenience!
